inc/formanswer.class.php

```diff
@@ -57,6 +57,7 @@ class PluginFormcreatorFormAnswer extends CommonDBTM
 
    // Values choosen to not conflict with status of ticket constants
    // @see PluginFormcreatorIssue::getNewStatusArray
+   const WAITING_APPROVAL = 101;
    const STATUS_WAITING = 101;
    const STATUS_REFUSED = 102;
    const STATUS_ACCEPTED = 103;
```

Comment:
2 constants shouldn't have the same value.

I misspoke, I was thinking of renaming the 3 constants, as STATUS_... is not explicit enough.

```suggestion
   const APPROVAL_WAITING = 101;
   const APPROVAL_REFUSED = 102;
   const APPROVAL_ACCEPTED = 103;
```

Reply:
I'm not sure a new value is the solution here (I'll let @btry confirm that) but if we add a new constant it should have the same prefix name `STATUS_XXX`